src/explore-education-statistics-frontend/src/modules/table-tool/components/TimePeriodDataTable.tsx

```diff
@@ -60,7 +62,8 @@ const TimePeriodDataTable = ({
       columns: timePeriods,
       rows: indicators,
     });
-  }, [columnGroups, rowGroups, timePeriods, indicators]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why the need for disabling lint here?

Reply:
Originally it was mostly to stop an infinite re-rendering of the component as the dependencies the lint wants will always cause the effect to fire (`columnGroups` and `rowGroups` are always different on each re-render). Thinking about it though, I'm going to try and see if there's a better way of doing this.